src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/service/CommunityApiService.kt

```diff
@@ -15,4 +19,13 @@ class CommunityApiService(
 
   fun getConvictionsWithSentences(crn: String): List<Conviction> =
     communityApiClient.getConvictions(crn).filterNot { it.sentence == null }.map { Conviction.from(it) }
+
```

Comment:
I'm not 100% sure these methods belong on this, although they might.
CommunityApiService was previously an n-tier style service layer on top of the Client, now it is also doing transformations on data retrieved directly from the client.
I know it's a lot of work and definitely not one for this PR but how about a single getRegistrations method which returns these three transformed properties on the object?

Reply:
Yes it could be a domain object.